ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -914,80 +859,138 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
+            </FlexRow>
+          </WorkspaceEditSection>
+
+            {/* TextBox: scientific approaches section*/}
+            <WorkspaceEditSection
+                header={researchPurposeQuestions[3].header}
+                description={researchPurposeQuestions[3].description} index='2.2'>
+              <TextArea style={{height: '15rem', resize: 'none', width: '50rem', borderRadius: '3px 3px 0 0',
+                borderColor: colorWithWhiteness(colors.dark, 0.5)}}
+                        id='scientificApproach'
+                        name='scientificApproach'
+                        value={this.state.workspace.researchPurpose.scientificApproach}
+                        onChange={v => this.updateResearchPurpose('scientificApproach', v)}/>
+              <FlexRow style={styles.textBoxCharRemaining}>
+                {this.state.workspace.researchPurpose.scientificApproach &&
+                <div>{1000 - this.state.workspace.researchPurpose.scientificApproach.length}
+                characters remaining</div>}
+                {!this.state.workspace.researchPurpose.scientificApproach &&
+                <div>1000 characters remaining</div>}
+              </FlexRow>
+            </WorkspaceEditSection>
+
+            {/*TextBox: anticipated findings from the study section*/}
+            <WorkspaceEditSection header={researchPurposeQuestions[4].header}
+                                  description={researchPurposeQuestions[4].description} index='2.3'>
+              <TextArea style={{height: '15rem', resize: 'none', width: '50rem', borderRadius: '3px 3px 0 0',
+                borderColor: colorWithWhiteness(colors.dark, 0.5)}}
+                        id='anticipatedFindings'
+                        name='anticipatedFindings'
+                        value={this.state.workspace.researchPurpose.anticipatedFindings}
+                        onChange={v => this.updateResearchPurpose('anticipatedFindings', v)}/>
+              <FlexRow style={styles.textBoxCharRemaining}>
+                {1000 - this.state.workspace.researchPurpose.anticipatedFindings.length} characters remaining
+              </FlexRow>
+            </WorkspaceEditSection>
+          </FlexColumn>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header={researchPurposeQuestions[3].header}
-                              description={researchPurposeQuestions[3].description} required>
-          <TextArea value={this.state.workspace.researchPurpose.anticipatedFindings}
-                    onChange={v => this.updateResearchPurpose('anticipatedFindings', v)}/>
+
+          {/*disseminate  research Section */}
+        <WorkspaceEditSection header={researchPurposeQuestions[5].header}
+                              description={researchPurposeQuestions[5].description} style={{width: '50rem'}} index='3.'>
+          <FlexRow>
+            <FlexColumn style={{flex: '1 1 0', marginLeft: '1rem'}}>
```

Comment:
This looks like the same styles as the FlexColumn below, can we move that to a style object?